content/en/blog/2020/large-scale-security-policy-performance-tests/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Large Scale Security Policy Perfomance Tests
+subtitle: How does having security policies effect latency of requests
+description: How does having security policies effect latency of request.
+publishdate: 2020-09-02
+attribution: "Michael Eizaguirre (Google)"
+keywords: [test,security policy]
+---
+
+## Overview
+
+Istio has a wide range of Security policies which can be easily configured into systems of services. As the number of Security policies being applied increases it is important to understand how much latency these Security policies are adding to the system.
+
+This blog post goes over common Security policies use cases and how the number of Security policies or the number of specific rules in a Security policy can effect the overall latency of requests.
+
+## Setup
+
+There are a wide range of Security policies and many more combinations of those policies. We will go over 6 test cases that were determined to be more common/valuable to users.
+
+The following test cases are run in an environment which consists of a fortio client sending requests to a fortio server, except of that of a the baseline which has no Envoy sidecars deployed.
+{{< image width="55%" ratio="45.34%"
+    link="./istio_setup.svg"
+    caption="Environment setup"
+    >}}
+
+The policies that are applied are made to not match with any request or only the very last rule, this makes sure that the RBAC filter has to go through all the rules and policies. This forces the system to never short circuit (when the RBAC filter matches before viewing all the policies). Even though this is not necessarily what will happen in your individual system, this gives data of the worst possible performance of each test case.
+
+## Test cases
+
+1. MTLS STRICT vs plaintext.
+
+1. A single authorization policy with a variable number of principal rules. For the RBAC filter to activate this rule a peer authentication policy must also be applied to the system.
+
+1. Authorization policy with a variable number of `requestPrincipal` rules. For the RBAC filter to activate this rule a `requestAuthentication` policy must also be applied to the system.
```

Comment:
Is this a single policy or variable number of policies?

Reply:
It's a bit misleading to say `For the RBAC filter to activate this rule`, can we update to say that the `requestPrincipal` is depending on the `RequestAuthentication` policy, no need to mention RBAC filter.

Reply:
Sounds good updated it